server/RoboFile.php

```diff
@@ -38,7 +38,7 @@ public function deployPantheon($branchName = 'master') {
 
     if ($result->getMessage()) {
       $this->yell($result->getMessage());
-      throw new Exception('The GitHub working directory is dirty. Please commit any pending changes or add to .gitignore.');
+      // throw new Exception('The GitHub working directory is dirty. Please commit any pending changes or add to .gitignore.');
```

Comment:
We should solve it in a different way, we should keep the working directory clean simply.

Reply:
This was never meant to be commited. Reverted.